File: 11.cs
int k, power, divisor;
double dividend, total = 0;

Console.ForegroundColor = ConsoleColor.Green;
Console.Write("\n4 * (-1)^k + 1");
Console.Write("\n    ----------  =");
Console.Write("\n      2k - 1");
Console.Write("\n4 * (");

for (k = 1; k <= 1000000; k++)
{
    power = k + 1;
    dividend = Math.Pow(-1, power);
    divisor = (2 * k) - 1;
    total += 4 * (dividend / divisor);

    if (dividend > 0) Console.Write(" + " + (int)dividend + " / " + divisor);
    else
    {
        Console.Write(" -1 / " + divisor);
    }
}

string s = String.Format("{0:0.00}", total);
Console.Write(") = " + s);
Console.ForegroundColor = ConsoleColor.White;
Console.WriteLine("\n\n\n\n\n");

return 0;

File: 11.py
class Colors:
    
    green = "\033[92m"
    white = "\033[0m"

print(Colors.green + "\n4 * (-1)^k + 1\n    ----------  ="
      + "\n      2k - 1\n4 * (", end='')

total = 0
for i in range(1, 1001):
    
    power = i + 1
    dividend = pow(-1, power)
    divisor = (2 * i) - 1
    total += 4 * (dividend / divisor)

    if (dividend > 0):
        print(" + {0} / {1}".format(dividend, divisor), end='')
    else:
        print(" - 1 / {0}".format(divisor), end='')
    
    i += 1

print(") = {0}".format(round(total, 2)) + Colors.white)
print("\n\n\n\n\n\n")
